lib/Twig/Extensions/Extension/I18n.php

```diff
@@ -17,7 +28,7 @@ class Twig_Extensions_Extension_I18n extends Twig_Extension
      */
     public function getTokenParsers()
     {
-        return array(new Twig_Extensions_TokenParser_Trans());
+        return array(new Twig_Extensions_TokenParser_Trans($singularFunc = $this->singularFunc, $pluralFunc = $this->pluralFunc));
```

Comment:
You can take the assignments out, these don't do anything for you so are basically a no-op.
It is not the cause for the PHP7 fail btw.